docs/content/preview/explore/connection-manager/connection-mgr-ysql.md

```diff
@@ -28,7 +28,7 @@ YugabyteDB includes a built-in connection pooler, YSQL Connection Manager, which
 YSQL Connection Manager is currently not supported for [YugabyteDB Anywhere](../../../yugabyte-platform/) and [YugabyteDB Managed](../../../yugabyte-cloud/).
 {{< /note >}}
 
-<!-- ----- Add Diagram---- -->
+![Connection manager](/images/explore/connection-manager.png)
```

Comment:
Diagram looks fine, but I don't know what these objects are really. Post master? PG backend is doing what?
Surrounding text doesn't explain what's going on